app/controllers/commodities_controller.rb

```diff
@@ -30,19 +31,29 @@ def set_prev_next(commodity)
     ]
   end
 
-  def fetch_commodity_from_xi
-    @xi_commodity = TradeTariffFrontend::ServiceChooser.with_source(:xi) do
-      CommodityPresenter.new(Commodity.find(params[:id], query_params))
+  def fetch_commodity
```

Comment:
```suggestion
  def fetch_commodities
```

Reply:
I used the singular because it's technically only 1 commodity in up to 2 versions.

But, if you think it's more descriptive in the plural, should I also change `fetch_heading` to `fetch_headings` in the headings controller?

Reply:
I'm not super picky :)